src/runtime/markdown-parser/handler/utils.ts

```diff
@@ -20,7 +20,7 @@ export function parseThematicBlock (lang: string) {
 
   const language = lang.replace(/[{|[](.+)/, '').match(/^[^ \t]+(?=[ \t]|$)/)
   const highlightTokens = lang.match(/{([^}]*)}/)
-  const filenameTokens = lang.match(/\[([^\]]*)\]/)
+  const filenameTokens = lang.match(/\[(.*)\]/) /** Allow ']' in filename. @see https://regex101.com/r/Hm0t5m/1 */
```

Comment:
```suggestion
  const filenameTokens = lang.match(/\[((\\]|[^\]])*)\]/)
```

Reply:
And because of GitHub, I'm not allowed to suggest changes out of some range:

Change for L29:

```diff
-    filename: Array.isArray(filenameTokens) && filenameTokens[1] ? filenameTokens[1] : undefined,
+    filename: Array.isArray(filenameTokens) && filenameTokens[1] ? filenameTokens[1].replace(/\\]/g, ']') : undefined,
```

Reply:
Question is... is `\` or `\\` in Markdown a backslash?
Do we have to write `\\]` or `\]` in the markdown file?

Reply:
For test, I think that we need to extract these utilities into simple function to be able to unit tests them.